pkg/scheduler/framework/plugins/interpodaffinity/plugin.go

```diff
@@ -52,6 +53,15 @@ func (pl *InterPodAffinity) Name() string {
 	return Name
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable
+func (pl *InterPodAffinity) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		{Resource: framework.Pod, ActionType: framework.Delete},
```

Comment:
PodAffinity is a bit different than other plugins, we should consider whether the event change can make a failed pod schedulable. So:

- PodDelete is one, b/c if a pod failed due to violating existing Pod's anti-affinity constraints, deleting an existing pods can make it schedulable
- PodUpdate can also be helpful, b/c an existing Pod's label can be changed (removed), so an unschedulable Pod may become schedulable
- PodAdd can also be helpful, b/c a unschedulable Pod may failed by pod-affinity constraints, and an added Pod may make it schedulable.

So we should put `framework.All` here.

Reply:
I've replaced `framework.Pod` with `framework.All` as suggested.